src/main/java/ch/njol/skript/effects/EffSendTitle.java

```diff
@@ -34,13 +34,21 @@
 import ch.njol.util.Kleenean;
 
 @Name("Title - Send")
-@Description({"Sends a title/subtitle to the given player(s) with optional fadein/stay/fadeout times for Minecraft versions 1.11 and above.",
-		"Note: if no input is given for the title/subtitle or the times," +
-		"it will keep the ones from the last title sent, use the <a href='effects.html#EffResetTitle'>reset title</a> effect to restore the default values."})
-@Examples({"send title \"Competition Started\" with subtitle \"Have fun, Stay safe!\" to player for 5 seconds",
-		"send title \"Hi %player%\" to player", "send title \"Loot Drop\" with subtitle \"starts in 3 minutes\" to all players",
-		"send title \"Hello %player%!\" with subtitle \"Welcome to our server\" to player for 5 seconds with fadein 1 second and fade out 1 second",
-		"send subtitle \"Party!\" to all players"})
+@Description({
+	"Sends a title/subtitle to the given player(s) with optional fadein/stay/fadeout times for Minecraft versions 1.11 and above. ",
+	"",
+	"If you're sending only the subtitle, it will be shown only if there's a title displayed at the moment, otherwise it will " +
+	"be sent with the next title. To show only the subtitle, use: <code>send title \" \" with subtitle \"yourtexthere\" to player</code>.",
+	"",
+	"Note: if no input is given for the times, it will keep the ones from the last title sent, " +
+	"use the <a href='effects.html#EffResetTitle'>reset title</a> effect to restore the default values."
+})
+@Examples({
+	"send title \"Competition Started\" with subtitle \"Have fun, Stay safe!\" to player for 5 seconds",
+	"send title \"Hi %player%\" to player", "send title \"Loot Drop\" with subtitle \"starts in 3 minutes\" to all players",
```

Comment:
One example per line \:)
```suggestion
	"send title \"Hi %player%\" to player",
	"send title \"Loot Drop\" with subtitle \"starts in 3 minutes\" to all players",
```